matkono/HealthSupApp
# Doing a lot of things

## What this PR does

- Created patient feature and screens;
- Created disease feature and screens;
- Created settings feature and screens;
- Overlay to the patient homepage;
- Internet connection error functionality;
- Pop-up password recovery;
- Integration between patient, disease, settings and decision tree features;

## How to check if this PR is correct

- Check all the functionality at patient screens;
- Check the disease screen;
- Check all the functionality at settings screens;
- Click the 'Ajuda' button at patient homepage;
- Check the message error when trying to login without internet connection;
- See the pop-up when trying to recover the password;
- By clicking the 'Continuar' and 'Nova Consulta' buttons, you should be redirected to the decision tree page;

## Evidences: 

- Patient homepage:
![image](https://user-images.githubusercontent.com/44846533/97648037-83642f00-1a32-11eb-94a2-e2c763f7c946.png)

- Edit patient screen:
![image](https://user-images.githubusercontent.com/44846533/97647968-544dbd80-1a32-11eb-8cfb-9837723c81d0.png)

- New patient screen:
![image](https://user-images.githubusercontent.com/44846533/97647987-62034300-1a32-11eb-97e5-21c5e812fcee.png)

- Patient details:
![image](https://user-images.githubusercontent.com/44846533/97648007-6fb8c880-1a32-11eb-80a5-21334b94f7d7.png)

- Overlay patient homepage:
![image](https://user-images.githubusercontent.com/44846533/97648088-a7277500-1a32-11eb-8ffb-cea25061e0e1.png)

- Message error internet connection:
![image](https://user-images.githubusercontent.com/44846533/97648752-4d27af00-1a34-11eb-9ba8-835092c3c19d.png)

- Recover password screen:
![image](https://user-images.githubusercontent.com/44846533/97650648-f670a400-1a38-11eb-8f6e-a83fa0d992f7.png)

- Disease homepage:
![image](https://user-images.githubusercontent.com/44846533/97650191-e906ea00-1a37-11eb-918d-90b7b821ed69.png)

- Settings homepage:
![image](https://user-images.githubusercontent.com/44846533/97650248-050a8b80-1a38-11eb-8b03-3985c587f0b9.png)

- Account details screen:
![image](https://user-images.githubusercontent.com/44846533/97650304-29fefe80-1a38-11eb-9907-736c2e1f7001.png)

- Alter account data screen:
![image](https://user-images.githubusercontent.com/44846533/97650350-400cbf00-1a38-11eb-9852-8af091ec8c39.png)

- Account password change screen:
![image](https://user-images.githubusercontent.com/44846533/97650373-4b5fea80-1a38-11eb-988b-9a32d62b7a36.png)

- App details screen:
![image](https://user-images.githubusercontent.com/44846533/97650453-76e2d500-1a38-11eb-9ca2-e826e8b29628.png)

- Report problem screen:
![image](https://user-images.githubusercontent.com/44846533/97650678-0ab4a100-1a39-11eb-8e6e-86704ee931f3.png)

- Pop-up recovery password:
![image](https://user-images.githubusercontent.com/44846533/97650711-20c26180-1a39-11eb-9999-dd0b0758a2a8.png)
